SpatialGDK/Source/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -621,6 +621,17 @@ void USpatialActorChannel::UpdateSpatialPosition()
 
 void USpatialActorChannel::UpdateSpatialRotation()
 {
+	FRotator ActorSpatialRotation = Actor->GetActorRotation();
+
+	// Check that it's rotated sufficiently far to be updated.
+	const float SpatialRotationThreshold = 0.1f;  // 0.1 radian (~5.7 degrees)
+	FRotator RotationDelta = ActorSpatialRotation - LastSpatialRotation;
+	if (RotationDelta.Quaternion().AngularDistance(FQuat::Identity) < SpatialRotationThreshold)
```

Comment:
Couldn't you use `FRotator::Equals()` with a small tolerance parameter to make this simpler?

Reply:
Looks like this doesn't quite do the same thing. `FRotator::Equals()` checks to see if the difference in any individual component of the rotator (Yaw, Pitch, Roll) is larger than the tolerance parameter, which is much more fuzzy. Happy to change to that if you feel strongly about it, but it doesn't do the same thing.

That said, the current implementation is more expensive, so if it turns out to be a big CPU cycle bottleneck then happy to change to `FRotator::Equals()` later.